kuma/wiki/templates/wiki/document.html

```diff
@@ -387,10 +388,16 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
                       {{ datetimeformat(current_revision.created, format='datetime') }}
                     </div>
                   {% endif %}
+		  {% if is_admin %}
+		    <div class="contributors-sub">
+		      Document number : {{ document.id }}
```

Comment:
By convention, it would be great if we could remove the space between the word *number* and the colon.